root/statistics/Editors.js

```diff
@@ -0,0 +1,143 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {l_statistics as l} from '../static/scripts/common/i18n/statistics';
+import EditorLink from '../static/scripts/common/components/EditorLink';
+import {withCatalystContext} from '../context';
+import loopParity from '../utility/loopParity';
+
+import {formatCount} from './utilities';
+import StatisticsLayout from './StatisticsLayout';
+
+type EditorsStatsT = {|
+  +dateCollected: string,
+  +topEditors: $ReadOnlyArray<EditorStatT>,
+  +topRecentlyActiveEditors: $ReadOnlyArray<EditorStatT>,
+  +topRecentlyActiveVoters: $ReadOnlyArray<EditorStatT>,
+  +topVoters: $ReadOnlyArray<EditorStatT>,
+|};
+
+type EditorStatsTableProps = {|
+  +$c: CatalystContextT,
+  countLabel: string,
+  dataPoints: $ReadOnlyArray<EditorStatT>,
+  editorLabel: string,
+  tableLabel: string,
+|};
+
+type EditorStatT = {|
+  +count: number,
+  +editor: EditorT,
+|};
+
+const EditorStatsTable = withCatalystContext(({
+  $c,
+  countLabel,
+  dataPoints,
+  editorLabel,
+  tableLabel,
+}: EditorStatsTableProps) => (
+  <>
+    <h3>{tableLabel}</h3>
+    <table className="tbl">
+      <thead>
+        <tr>
+          <th className="pos">{l('Rank')}</th>
+          <th>
+            {editorLabel}
+            <div className="arrow" />
+          </th>
+          <th>
+            {countLabel}
+            <div className="arrow" />
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        {dataPoints.length > 0 ? (
+          dataPoints.map((editorStat, index) => (
+            <tr
+              className={loopParity(index)}
+              key={
+                editorStat.editor
+                  ? editorStat.editor.id
+                  : `missing-${index}`
+              }
+            >
+              <td className="t">{index + 1}</td>
+              <td><EditorLink editor={editorStat.editor} /></td>
+              <td className="t">{formatCount($c, editorStat.count)}</td>
+            </tr>
+          ))
+        ) : (
+          <tr className="even">
+            <td colSpan="3">{l('There is no data to display here.')}</td>
+          </tr>
+        )}
+      </tbody>
+    </table>
+  </>
+));
+
+const Editors = ({
+  dateCollected,
+  topEditors,
+  topRecentlyActiveEditors,
+  topRecentlyActiveVoters,
+  topVoters,
+}: EditorsStatsT) => (
+  <StatisticsLayout fullWidth page="editors" title={l('Editors')}>
```

Comment:
Tab labels are not localized for this page and some others.

Reply:
I know. I have a fix for that, I think it was in the next PR though